sda-commons-server-morphia/src/main/java/org/sdase/commons/server/morphia/MongoConfiguration.java

```diff
@@ -103,6 +120,9 @@ public MongoConfiguration setDatabase(String database) {
   }
 
   public String getOptions() {
+    if (StringUtils.isBlank(options) && StringUtils.isNotBlank(connectionString)) {
+      // TODO what should be returned?
```

Comment:
The [mongodb-operator is doing some magic](https://github.com/SDA-SE/mongodb-operator/blob/master/src/main/java/com/sdase/k8s/operator/mongodb/controller/tasks/util/ConnectionStringUtil.java#L33-L39) with URI to get the query part. I wonder if we actually need a value here when the connection string is defined. Seems [like URI can handle](https://github.com/SDA-SE/mongodb-operator/blob/master/src/test/java/com/sdase/k8s/operator/mongodb/controller/tasks/util/ConnectionStringUtilTest.java#L11-L35) the comma in the host part.

Reply:
Isn't your example in the mongodb-operator prone to errors? The magic is not really clear to me.